src/Catalyst.Core.Modules.Consensus/Deltas/DeltaCache.cs

```diff
@@ -46,18 +51,34 @@ public class DeltaCache : IDeltaCache, IDisposable
         private readonly Func<MemoryCacheEntryOptions> _entryOptions;
         public Cid GenesisHash { get; set; }
 
-        public static string GetLocalDeltaCacheKey(CandidateDeltaBroadcast candidate)
-        {
-            return nameof(DeltaCache) + "-LocalDelta-" + MultiBase.Encode(candidate.Hash.ToByteArray(), "base32");
-        }
+        public static string GetLocalDeltaCacheKey(CandidateDeltaBroadcast candidate) { return nameof(DeltaCache) + "-LocalDelta-" + MultiBase.Encode(candidate.Hash.ToByteArray(), "base32"); }
+
+        public static Address TruffleTestAccount = new Address("0xb77aec9f59f9d6f39793289a09aea871932619ed");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4979613)